regolith/schemas.py

```diff
@@ -184,21 +184,22 @@
                      'city': 'New York',
                      'country': 'USA',
                      'departments': {
-                         'physics': 'Department of Physics',
-                         'chemistry': 'Department of Chemistry',
-                         'apam': 'Department of Applied Physics'
-                                 'and Applied Mathematics',
-                         'cheme': 'Department of Chemical'
-                                  'Engineering',
-                         'cs': 'Computer Science Department',
-                         'eaee': 'Department of Earth and'
-                                 'Environmental Engineering'
+                         'physics': {'name': 'Department of Physics',
```

Comment:
Due to this change I think you need to change the builders which rely on this, since they are expecting `dict['physics']` to be a string, but now it is a dict (I think you need `dict['physics']['name']`).

Reply:
yup, this will break the builder, but I can fix it again when the new schema is fixed and pulled in.

Reply:
It might be nice if we could not break the builder. If you don't want to handle it in this PR, do you want to merge this PR with the other preslist PR?

Reply:
You could pull this branch into your other branch, you could rebase one onto the other, you could issue a PR into yourself, you could copy paste this code into the other branch (although I don't suggest that so much), there are a couple options. Whichever one works the best for you.

Reply:
you mean there is no "merge PRs" button on GitHub?  :(

Reply:
Not to my knowledge @scopatz thoughts?

Reply:
just kidding.....I will rebase and close the other PR